drivers/staging/media/rpivid/rpivid_video.c

```diff
@@ -27,6 +27,8 @@
 
 #define RPIVID_MIN_WIDTH	16U
 #define RPIVID_MIN_HEIGHT	16U
+#define RPIVID_DEFAULT_WIDTH	1920U
+#define RPIVID_DEFAULT_HEIGHT	1088U
```

Comment:
Why 1088? The bare literal in the code was 1080.

Reply:
Because I'm expecting an HD stream to be 1088 high cropped to 1080.  I'm just picking what I hope to be a better number.  In actual use the default will probably be never used and the sizes will be explicitly set by the user or inferred from the SPS.